src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -157,6 +158,7 @@ final class RiskProvider: RiskProviding {
 
 	private let store: Store
 	private let appConfigurationProvider: AppConfigurationProviding
+	private let appFeatureProvider: AppFeatureProviding
```

Comment:
This is only used for the DeviceTimeCheck right? 
Why not store the DeviceTimeCheck directly during init?

Reply:
Good point, I'll try it.

Reply:
I did some more refactoring to get rid of a dependency cycle, please have another look